frontend/src/views/Course.vue

```diff
@@ -73,10 +85,51 @@ export default {
       courseInfo: 'course'
     })
   },
+  methods: {
+    update() {
+      this.$store.dispatch('course/getCourse', this.code)
+      this.refreshQuestions()
+      this.refreshReviews()
+    },
+    refreshQuestions() {
+      this.$store.dispatch('course/getQuestions',
+        {
+          id: this.code,
+          pageNumber: this.questionPage
+        })
+    },
+    refreshReviews() {
+      this.$store.dispatch('course/getReviews',
+        {
+          id: this.code,
+          pageNumber: this.reviewPage
+        })
+    },
+    nextPage (feedType) {
```

Comment:
since you'll presumably extend Feed to work with answers, and update these functions according, you might want to make a function factory to generate nextPage/prevPage functions for answers, reviews and questions.